exetera/core/fields.py

```diff
@@ -162,8 +177,18 @@ def apply_index(self, index_to_apply, dstfld=None):
         raise NotImplementedError("Please use apply_index() on specific fields, not the field base class.")
 
 
+# Field arrays
+# ============
+
+
 class ReadOnlyFieldArray:
     def __init__(self, field, dataset_name):
+        """
+        Construct a readonly FieldArray which used as the wrapper of data in Fields (apart from IndexedStringFields).
```

Comment:
Construct a ReadOnlyFieldArray instance. This class is an implementation detail, used to access data in non-indexed fields deriving from HDF5Field. As such, instances of ReadOnlyFieldArray should only be created by the fields themselves, and not by users of ExeTera.